Formula/jumanpp.rb

```diff
@@ -0,0 +1,24 @@
+class Jumanpp < Formula
+  desc "Japanese Morphological Analyzer based on RNNLM"
+  homepage "http://nlp.ist.i.kyoto-u.ac.jp/EN/index.php?JUMAN%2B%2B"
+  url "http://lotus.kuee.kyoto-u.ac.jp/nl-resource/jumanpp/jumanpp-1.01.tar.xz"
+  sha256 "0d587416a3eb7123638f9c1e30a649b72dfb483448839168dcb48be572c5919a"
+
+  depends_on "boost-build" => :build
+  depends_on "gperftools" => :build
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    # Currently there is a bug in Makefile.
+    # `echo "" | src/jumanpp -D juman-resources` is too slow so that make process exits with error code.
+    # Preventing this bug, disable parallel build
+    system "make", "-j1"
+    system "make", "install"
+  end
+
+  test do
+    system "jumanpp", "-h"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Additionally, use `system "#{bin}/jumanpp"`. Thanks!